source/particle/output/hdf5.cc

```diff
@@ -293,7 +292,6 @@ namespace aspect
         return output_path_prefix;
 #else
         (void) property_information;
-        (void) particles;
```

Comment:
replace this by the new unused argument, ``particle_handler``

Reply:
ah, now I see what they do :)